GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -67,7 +67,7 @@ protected override void OnRuntimeLoad(EventArgs e)
 
             _NO_TRANSLATE_To.Text = AppSettings.DefaultCloneDestinationPath;
 
-            if (url.IsNotNullOrWhitespace())
+            if (GitModule.PathIsUrl(url) && GitModule.IsValidGitWorkingDir(url))
```

Comment:
I think `url` variable can't be url and path at the same time, `if` will never work.